File: benchmark/pi.py
# pi.py — π via a telescoped Leibniz series; the Python twin of pi.alc.
acc = 0.0
k = 1
while k < 40000000:
    acc += 4.0 / k - 4.0 / (k + 2)
    k += 4
print(f"{acc:g}")

File: benchmark/pi.c
/* pi.c — π via a telescoped Leibniz series; the native twin of pi.alc.
   Same terms in the same order, so the IEEE double result is identical. */
#include <stdio.h>

int main(void) {
  double acc = 0.0;
  for (long k = 1; k < 40000000L; k += 4)
    acc += 4.0 / (double)k - 4.0 / (double)(k + 2);
  printf("%g\n", acc);
  return 0;
}
